.github/workflows/continuous-tests.yml

```diff
@@ -50,6 +50,33 @@ jobs:
         with:
           name: coverage
           path: coverage/
+  maria:
+    name: MariaProvider
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Project
+        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b # tag=v3
+      - name: Use Node.js v16
+        uses: actions/setup-node@2fddd8803e2f5c9604345a0b591c3020ee971a93 # tag=v3
+        with:
+          node-version: 16
+      - name: Shutdown MySQL
```

Comment:
Why is the mysql database being shutdown and then started?

Reply:
It's the recommended method in the Maria documentation, afaik GitHub actions have an already running MySQL server that needs to be stopped before starting a mariadb